src/averageAges.js

```diff
@@ -20,6 +20,16 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const men = people.filter(person => century
+    ? person.sex === 'm' && Math.ceil(person.died / 100) === century
+    : person.sex === 'm');
```

Comment:
you check 'century' on every iteration. You can check the 'century' only 1 time and choose what filter to apply (the same in next functions)